samples/CloudClipboard/SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.2 BLOCK -->
+
+## Security
```

Comment:
This exists in the repository root;  there shouldn't be another copy here.

Reply:
Removed